From 4da69c73a7cccc01300e5feafd2e632a57c6d3ff Mon Sep 17 00:00:00 2001
From: Ulrich Hecht <uli@basicengine.org>
Date: Wed, 4 Oct 2023 17:45:18 +0200
Subject: [PATCH 042/184] 
 linux-mainline-and-mali-allwinner-h6-kernel/misc.616/patches/h616-higher-temps-v6.1.patch

---
 arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
index 59e2ad67b..02a92a4e8 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
@@ -1225,13 +1225,13 @@ cpu-thermal {
 
 			trips {
 				cpu_alert: cpu-alert {
-					temperature = <85000>;
+					temperature = <95000>;
 					hysteresis = <2000>;
 					type = "passive";
 				};
 
 				cpu-crit {
-					temperature = <100000>;
+					temperature = <105000>;
 					hysteresis = <0>;
 					type = "critical";
 				};
-- 
2.39.2

